custom_components/sagemcom_fast/options_flow.py

```diff
@@ -0,0 +1,36 @@
+"""Options flow for Sagemcom integration."""
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_SCAN_INTERVAL
+import voluptuous as vol
+
+SCAN_INTERVAL = 10
```

Comment:
```suggestion
DEFAULT_SCAN_INTERVAL = 10
```
Could you rename it and move it to `const.py`?